views/show-conversation.pug

```diff
@@ -1,9 +1,13 @@
 html
   head
-    title CONVERSATION BETWEEN #{participants[0].toUpperCase()} AND #{participants[1].toUpperCase()}
+    title #{participants[0].toUpperCase()}, #{participants[1].toUpperCase()}
```

Comment:
i'm realizing that it makes a lot more sense to just use css to control whether or not we want this uppercased -- and it's safer, since we won't be calling a function on a possibly null value. could you change this bit so that we don't call `.toUpperCase()` on either participant?

Reply:
yes! i dont know much java but is this what you mean:
#{participants[0]}, #{participants[1]}